README.md

```diff
@@ -76,9 +76,51 @@ Or you can use a callback
         }
     });
 
-## Documentation
+## API Documentation
 
-For more details about the current release, please consult [our documentation](https://aerogear.org/docs/unifiedpush/).
+### Class: Sender
+
+The Sender Class,  It is an `EventEmitter`
+
+### new Sender(settings)
+
+* `settings` Object
+    * `url` String - The URL of the Unified Push Server.
+    * `applicationId` String - The id of an Application from the Unified Push Server
+    * `masterSecret` String - The master secret for that Application
+
+### sender.send([message], [options], [callback])
+
+* `message` Object
+    * `alert` String - message that will be displayed on the alert UI element
+    * `actionCategory` String -  the identifier of the action category for the interactive notification
```

Comment:
the new apple keys are in this branch,  https://github.com/lholmquist/aerogear-unifiedpush-nodejs-client/tree/applie-keys

once this gets in,  then i'll send that one